test/functional/validation-options.spec.ts

```diff
@@ -1,5 +1,5 @@
 import "es6-shim";
-import {Contains, Matches, MinLength, ValidateNested} from "../../src/decorator/decorators";
+import {Contains, Matches, MinLength, ValidateNested, IsDefined} from "../../src/decorator/decorators";
```

Comment:
would be nice to remove relative paths in imports here

Reply:
As it's a forked project, we're trying to make as few changes as possible.

Reply:
👍